### DISCUSSION TITLE
Is there an outdoor battery-powered camera that supports WiFi and is compatible with Frigate?

### DISCUSSION THREAD

**Discussion by AaronM04** [blakeblackshear/frigate#3692] [Ask A Question] [answered]
I was hoping the [AlkiVision Outdoor C2](https://amazon.com/Security-Waterproof-Rechargeable-Surveillance-Detection/dp/B092YVSL2Q) would be compatible with Frigate, since the product question about RTSP support was answered "yes" by three people, but that doesn't seem to actually be supported. All ports are closed on the device.

Should I just return these and buy another one? Any recommendations? Thanks in advance!

**Comment by NickM-27** [comment, marked as answer]
Have you tried to find settings in a web interface? Likely just needs to be enabled

**Reply by AaronM04** [reply]
It doesn't appear to have a web interface, only an app (Adorcam). There are no options in the app for this.

**Reply by NickM-27** [reply]
I just realized it's battery powered, no camera can have a battery big enough to support rtsp (24/7 streaming)

**Reply by AaronM04** [reply]
That's unfortunate.

**Reply by AaronM04** [reply]
What if the battery requirement is removed? Are there compatible cameras that use WiFi but an external power source?

**Reply by NickM-27** [reply]
> What if the battery requirement is removed? Are there compatible cameras that use WiFi but an external power source?

Yes, there are plenty of WiFi mains powered cameras that have rtsp streams
